src/Security/Security.php

```diff
@@ -953,6 +998,8 @@ public static function findAnAdministrator()
                 ->add($member);
         }
 
+        static::singleton()->extend('afterFindAdministrator');
```

Comment:
Explicitly _not_ passing in the member, to prevent any malicious code. Although, those kind of modules would be called out pretty quick I suspect